src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/CombinationController.php

```diff
@@ -572,6 +572,7 @@ private function formatListForPresentation(CombinationListForEditing $combinatio
                 'quantity' => $combination->getQuantity(),
                 'is_default' => $combination->isDefault(),
                 'image_url' => $combination->getImageUrl() ?: $fallbackImageUrl,
+                'eco_tax' => $combination->getEcoTax(),
```

Comment:
```suggestion
                'eco_tax' => (string) $combination->getEcoTax(),
```

Once it's been turned into a DecimalNumber